python/TestHarness/pbs_submit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#PBS -N <JOB_NAME>
+#PBS -l select=<CHUNKS>:ncpus=<NCPUS_PER_CHUNK>:mpiprocs=<MPI_PROCS_PER_CHUNK>
+#PBS -l place=<PLACE>
+#PBS -l walltime=<WALLTIME>
+#PBS -W Hold_Types=u
+<PBS_PREREQ>
+<PBS_PROJECT>
+<SOFT_LINK1>
+<QUEUE>
+<COMBINE_STREAMS>
+cd <PBS_O_WORKDIR>
+
+JOB_NUM=${PBS_JOBID%\.*}
+<SOFT_LINK2>
+
+export MV2_ENABLE_AFFINITY=0
+
+mpiexec <COMMAND>
+
+<SOFT_LINK3>
```

Comment:
That's fine, we might even simplify it a bit if we have our own in the TestHarness.